### PR TITLE
New endpoint integration to list transactions

### PR DESCRIPTION
**Pull request by arturoblanco**
Bitgo create a new andan easier enpoint to list wallet transactions and also to know if transaction is withdraw or deposit

### CODE REVIEW

File: lib/bitgo_client/v2.rb [hunk 1]
```diff
@@ -60,8 +64,18 @@ def transaction(wallet_id, transaction_id, coin_code: :tbtc, logger: nil)
       client.request("#{base_path}/#{coin_code}/wallet/#{wallet_id}/tx/#{transaction_id}", logger: logger)
     end
 
-    def fee(coin_code: :tbtc, logger: nil)
-      client.request("#{base_path}/#{coin_code}/tx/fee", logger: logger)
+    def transfers(wallet_id, coin_code: :tbtc, logger: nil, limit: 25, prev_id: nil, all_tokens: nil)
+      query_string = build_query_string(
+        limit:     limit,
+        prevId:    prev_id,
+        allTokens: all_tokens
```
Comment (rafaeltortella) [inline lib/bitgo_client/v2.rb:71]: why prevId and allToken are in camel case ? or they shoulded be "prev_id" and "all_token" ?

**Review by bsoares — APPROVED** [review]

File: lib/bitgo_client/v2.rb (hunk 1 above)
Comment (arturoblanco) [reply lib/bitgo_client/v2.rb:71]: Those are query string params for Bitgo requests

Comment (rafaeltortella) [reply lib/bitgo_client/v2.rb:71]: 👍

**Comment by rafaeltortella** [comment]
👍

**Pull request merged**